File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/UnpairCommand.java
package com.matter.controller.commands.pairing;

import chip.devicecontroller.ChipDeviceController;
import com.matter.controller.commands.common.CredentialsIssuer;

public final class UnpairCommand extends PairingCommand {
  public UnpairCommand(ChipDeviceController controller, CredentialsIssuer credsIssue) {
    super(controller, "unpair", PairingModeType.NONE, PairingNetworkType.NONE, credsIssue);
  }

  @Override
  protected void runCommand() {}
}

File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/UnpairCommand.kt
/*
 * SPDX-FileCopyrightText: 2023 Project CHIP Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 */
package com.matter.controller.commands.pairing

import chip.devicecontroller.ChipDeviceController
import com.matter.controller.commands.common.CredentialsIssuer

class UnpairCommand(controller: ChipDeviceController, credsIssue: CredentialsIssuer?) :
  PairingCommand(controller, "unpair", credsIssue, PairingModeType.NONE, PairingNetworkType.NONE) {
  override fun runCommand() {}
}
